src/components/pages/order/OrderPage.js

```diff
@@ -5,18 +5,24 @@ import { theme } from '../../../theme'
 import logoOrange from '../../../assets/logo-orange.png'
 import { fakeMenu2 } from '../../../fakeData/fakeMenu'
 import MenuCard from '../reusable-ui/MenuCard'
+import { useState } from 'react'
 
 export default function OrderPage() {
 
+    const [menuFake, setMenuFake] = useState(fakeMenu2)
```

Comment:
```suggestion
    const [menu, setMenu] = useState(fakeMenu2)
```

Quand le est **défini**, il s'appelle bien "menu" mais il est **initialisé** avec un jeu de donné fake.

Reply:
modification effectué